tests/integration-test.sh

```diff
@@ -20,12 +20,10 @@ gsutil version -l
 
 dpkg -l > package.list
 
-echo $DRONE_BUILD_URL
-echo $DRONE_BUILD_DIR
-echo $DRONE_PULL_REQUEST
-
 if [ $DRONE_BRANCH = "master" ] && [ $DRONE_REPO = "vmware/vic" ]; then
     pybot --removekeywords TAG:secret tests/test-cases
+elif grep -q "\[full ci\]" <(drone build info vmware/vic $DRONE_BUILD_NUMBER); then
```

Comment:
It is actually not skip-ci... it is skip ci.  Loc continues to use skip-ci and it doesn't work, per:
https://ci.vmware.run/vmware/vic/4344